tools/check_format.py

```diff
@@ -79,10 +86,15 @@ def isBuildFile(file_path):
 
 
 def checkFileContents(file_path):
+  pattern = '\.  '
   with open(file_path) as f:
     text = f.read()
-    if re.search('\.  ', text, re.MULTILINE):
-      printError("%s has over-enthusiastic spaces" % file_path)
+    if re.search(pattern, text, re.MULTILINE):
+      printError("%s has over-enthusiastic spaces:" % file_path)
+      regex = re.compile(pattern)
+      for i, line in enumerate(text.splitlines()):
```

Comment:
i see why `re` is useful here. can we abstract the find and output line number logic into a function to use it in both places?

Reply:
Sorry for the confusion but I also think we can use `in` instead of `re.search`. Is `re.search` still useful here?